sky/provision/gcp/constants.py

```diff
@@ -26,7 +26,7 @@
 # NOTE: `serviceAccountUser` allows the head node to create workers with
 # a serviceAccount. `roleViewer` allows the head node to run bootstrap_gcp.
 DEFAULT_SERVICE_ACCOUNT_ROLES = [
-    'roles/storage.objectAdmin',
+    'roles/storage.admin',
```

Comment:
The old sky/skylet/providers/gcp/config.py has this too, should it be changed (or should that dir be removed)?

Reply:
The dir should be removed now. Also, the AWS node provider should be removed as well #2792. We will handle that in another PR.